### PR TITLE
added new constructor for KinLine so we can use the Mu2eKinKal functions

### PR DESCRIPTION
**Pull request by sophiemiddleton**

### CODE REVIEW

**Comment by ryuwd** [comment]
CI failed due to an unrelated issue, fixed by #109

File: Trajectory/KinematicLine.cc [hunk 1]
```diff
@@ -76,6 +76,20 @@ namespace KinKal {
   KinematicLine::KinematicLine(Parameters const &pdata, KinematicLine const &other) : KinematicLine(other) {
     pars_ = pdata;
   }
+
```
Comment (brownd1978) [inline Trajectory/KinematicLine.cc:79]: Why are there data declarations in the middle of a .cc file?

File: Trajectory/LoopHelix.hh [hunk 2]
```diff
@@ -51,6 +51,7 @@ namespace KinKal {
       LoopHelix( Parameters const& pars, double mass, int charge, VEC3 const& bnom, TimeRange const& trange=TimeRange() );
       // copy payload and override the parameters; Is this used?
       LoopHelix(Parameters const& pdata, LoopHelix const& other);
+
```
Comment (brownd1978) [inline Trajectory/LoopHelix.hh:54]: Why did you add a blank line here?

Comment (sophiemiddleton) [reply Trajectory/LoopHelix.hh:54]: I will remove

File: Trajectory/KinematicLine.cc (hunk 1 above)
Comment (sophiemiddleton) [reply Trajectory/KinematicLine.cc:79]: This looks like an accidental pasting

Comment (sophiemiddleton) [reply Trajectory/KinematicLine.cc:79]: This looks like I accidentally pasted some information from the header at some point

**Comment by sophiemiddleton** [comment]
I have resolved these issues
